addons/logistics_wirecutter/XEH_preInit.sqf

```diff
@@ -10,3 +10,29 @@ PREP(interactEH);
 PREP(isFence);
 
 ADDON = true;
+
+
+// Condition for Interaction
+DFUNC(Condition) = {
```

Comment:
These functions are all below the `ADDON` statement on line 12. That should not be the case, as this is signaling that this modules functions are all initialized. 

I'd suggest to instead move the `progressCheck` function into separate file and `PREP` it, and the `condition` and `statement` back to their original files.